golang-job-type/go_wrapper/server.go

```diff
@@ -19,39 +19,32 @@ func WrapAndServe(entrypoint EntrypointHandler) error {
 	// Serve endpoints at raw path (to facilitate debugging) and prefixed path (when accessed through PUB).
 	// Accept any version so that job can be called by its many version names ("latest", "1.x").
 	baseUrls := []string{
-		fmt.Sprintf("/pub/job/%s/{version}", jobName),
+		fmt.Sprintf("/pub/job/%s/:version", jobName),
 		"",
 	}
-
-	router := mux.NewRouter()
+    
+    gin.SetMode(gin.ReleaseMode) //Hide debug routings
+	router := gin.New()
+    router.Use(gin.Recovery())
 
 	for _, baseUrl := range baseUrls {
 
-		router.HandleFunc(baseUrl+"/api/v1/perform", performHandler)
-		router.HandleFunc(baseUrl+"/health", HealthHandler)
-		router.HandleFunc(baseUrl+"/live", LiveHandler)
-		router.HandleFunc(baseUrl+"/ready", ReadyHandler)
+		router.GET(baseUrl+"/api/v1/perform", performHandler)
+        router.GET(baseUrl+"/health", HealthHandler)
+        router.GET(baseUrl+"/live", LiveHandler)
+        router.GET(baseUrl+"/ready", ReadyHandler)
+        router.GET(baseUrl+"/metrics", MetricHandler())
 		MountOpenApi(router, baseUrl)
 	}
 
-	loggingMiddleware := func(next http.Handler) http.Handler {
-		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			log.Info("Request", log.Ctx{
-				"method": r.Method,
-				"uri":    r.RequestURI,
-				"ip":     r.RemoteAddr,
-			})
-			next.ServeHTTP(w, r)
-		})
-	}
-	router.Use(loggingMiddleware)
+	router.Use(gin.Logger())
```

Comment:
It doesn't seem to (see [these docs](https://github.com/gin-gonic/gin/blob/v1.9.0/gin.go#L304-L306)), I just placed the new logging solution where the old one once was. I guess it might be better to group the `gin.Logger()` with the `gin.Recovery()` a bit further up?